src/osd/osd_perf_counters.cc

```diff
@@ -69,10 +69,10 @@ PerfCounters *build_osd_logger(CephContext *cct) {
     "Histogram of operation latency (including queue time) + data read");
   osd_plb.add_time_avg(
     l_osd_op_r_process_lat, "op_r_process_latency",
-    "Latency of read operation (excluding queue time)");
+    "Latency of read operation (excluding queue time)", NULL, PerfCountersBuilder::PRIO_INTERESTING);
```

Comment:
Similar here, `op_r_process_latency` is changed to interesting but its counterpart, `op_w_process_latency` is left untouched and therefore remains at useful.  Regardless of the chosen priority, we need to be consistent.

Reply:
In this case we saw that `op_w_process_latency` wasn't being used in grafana/prometheus but we did see`op_r_process_latency`.